handlers/firewall.go

```diff
@@ -35,14 +35,8 @@ type wafResponse struct {
 }
 
 func checkWAF(url string) (wafResponse, error) {
-	fullURL := ""
-	if !strings.HasPrefix(url, "http") {
-		fullURL = "http://" + url
-	} else {
-		fullURL = url
-	}
-
-	resp, err := http.Get(fullURL)
+	// TODO(Lissy93): does this test require we set scheme to http?
+	resp, err := http.Get(url)
```

Comment:
It just needs some protocol, whether http or https. 
But that's now already covered by urlutils